DNN Platform/Website/admin/Skins/Logo.ascx.cs

```diff
@@ -58,6 +64,38 @@ protected override void OnLoad(EventArgs e)
                         {
                             this.imgLogo.ImageUrl = imageUrl;
                             logoVisible = true;
+                        }
+
+                        if (this.isSVG.GetValueOrDefault() == true)

```

Comment:
This entire process is going to need defensive coding around it due to the fact that the provided image, might not actually be an SVG image so you will get errors and other stuff as part of this.

Additionally I do not believe that fileInfo.PhysicalPath is actually safe for use with third-party folder providers.  (But need to check)

Reply:
I added a check to make sure the file at least has an 'svg' extension.  If the person uploading a non-SVG as a .svg file, well... we can't do everything for them...  They could just as easily select a logo file that has a .jpg extension that is not actually a .jpg.

Reply:
I don't know how the third party folder stuff works, so I'll need some help there.